pkg/test/spaceprovisionerconfig/spaceprovisionerconfig.go

```diff
@@ -24,6 +25,38 @@ func NewSpaceProvisionerConfig(name string, namespace string, opts ...CreateOpti
 	return spc
 }
 
+func NewEnabledValidTenantSPC(name string, namespace string, referencedToolchainCluster string, opts ...CreateOption) *toolchainv1alpha1.SpaceProvisionerConfig {
+	return NewSpaceProvisionerConfig(name, namespace,
+		append(opts,
+			Enabled(true),
+			WithReadyConditionValid(),
+			ReferencingToolchainCluster(referencedToolchainCluster),
+			WithPlacementRoles(PlacementRole("tenant")))...,
+	)
+}
+
+func NewValidTenantSPC(name string, namespace string, referencedToolchainCluster string, opts ...CreateOption) *toolchainv1alpha1.SpaceProvisionerConfig {
+	return NewSpaceProvisionerConfig(name, namespace,
+		append(opts,
+			WithReadyConditionValid(),
+			ReferencingToolchainCluster(referencedToolchainCluster),
+			WithPlacementRoles(PlacementRole("tenant")))...,
+	)
+}
+
+func NewEnabledValidSPC(name string, namespace string, referencedToolchainCluster string, opts ...CreateOption) *toolchainv1alpha1.SpaceProvisionerConfig {
+	return NewSpaceProvisionerConfig(name, namespace,
+		append(opts,
+			Enabled(true),
+			WithReadyConditionValid(),
+			ReferencingToolchainCluster(referencedToolchainCluster))...,
+	)
+}
```

Comment:
just going through the other PRs - if I'm not mistaken, all the SPCs are created only in host-operator namespace, so you could simplify it, drop the "namespace" parameter, and set the `HostOperatorNs` automatically

Reply:
IMO keeping it as a parameter would be a better option . It would be hidden, but clean i guess

Reply:
That's a good observation. I thought about it and I think it's best to just move these utility functions into `host-operator` and only leave the most generic function here, because it is also used in e2e tests.

The latest commit here and in https://github.com/codeready-toolchain/host-operator/pull/991 should have these functions moved.

Reply:
good idea, thanks for moving it & simplifying it :+1: